assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js

```diff
@@ -106,7 +106,15 @@ function PageBackgroundPanel({ selectedElements, pushUpdate }) {
       true
     );
     clearBackgroundElement();
```

Comment:
My intention was to make sure that this function, `clearBackgroundElement`, did just that [in the underlying implementation](https://github.com/google/web-stories-wp/blob/54304800b6c6f32836696d1bee8b40d5f0d23024/assets/src/edit-story/app/story/useStoryReducer/reducers/setBackgroundElement.js#L49-L50) - and adding this restriction [to the unit test](https://github.com/google/web-stories-wp/blob/54304800b6c6f32836696d1bee8b40d5f0d23024/assets/src/edit-story/app/story/useStoryReducer/test/clearBackgroundElement.js#L23).

Reply:
I considered this but it doesn't somehow seem like a clean way to do this, it doesn't seem like the action of detaching background element should be aware of what's being selected, no? It seems to be more of a UX choice for the panel that media element should be unselected and Page selected instead after detaching, not something that the reducer should depend on. Thoughts?

Reply:
The reducer already manages selection as well and is opinionated on this for many other operations, so I don't see an issue with it being the source of truth for this either.

Reply:
Ok, fair enough.